packages/glob/src/internal-glob-options.ts

```diff
@@ -0,0 +1,28 @@
+export interface IGlobOptions {
+  /**
+   * Indicates whether to follow symbolic links. Generally should be true
+   * unless deleting files.
+   *
+   * @default true
+   */
+  followSymbolicLinks?: boolean
+
+  /**
+   * Indicates whether directories that match a glob pattern, should implicitly
+   * cause all descendant paths to be matched.
+   *
+   * For example, given the directory `my-dir`, the following glob patterns
+   * would produce the same results: `my-dir/**`, `my-dir/`, `my-dir`
+   *
+   * @default true
+   */
+  implicitDescendants?: boolean
```

Comment:
I need to update the ADR. When I started thinking about support for multiple patterns (and negate patterns) I came to the conclusion `implicitDescendants` is a better name instead of `expandDirectories`.

The new name aligns better with how the property influences negate patterns.